### ISSUE TITLE
Log level 'log' items are still logged with 'warn' or 'error' logLevel

### ISSUE THREAD

**Issue by edtownend** [djhouseknecht/rollup-plugin-version-injector#18]
Hi, thanks for a great plugin!

The only thing I've got a problem with is that even with the logLevel set to 'warn' or 'error', I'm still getting the following logged to my console:

```
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [error] code not set in VIInjector called from injectIntoTags()
  [VI] [error] code not set in VIInjector called from injectIntoComments()
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] found and replaced [1] version tags in poductivity-client.entry.esm.js
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished
  [VI] [log  ] version-injector started with version "0.9.0"
  [VI] [log  ] version-injector finished

```

My config is as follows:

```js
versionInjector({
  injectInComments: {
    fileRegexp: /\.(js|css)$/,
    tag: 'Poductivity SDK version: {version} - Built {date}',
    dateFormat: 'mmmm d, yyyy HH:MM:ss',
  },
  logLevel: 'warn',
}),
```

The logLevel is definitely being read from the config, because if I change it to 'debug' I get way more logged.

I'm using version 1.3.0,  and the context is a stenciljs app.

Thanks!

**Comment by djhouseknecht** [comment]
Ahh, yeah that is too chatty. I have some other work to do in VI and will tune down the logging at that same time.

**Comment by djhouseknecht** [comment]
This will be fixed in v1.3.2. It may take several hours before travis builds and releases the new version. Let me know if you still need changes after installing the new version. Thanks for the report and for using verion-injector!

**Comment by edtownend** [comment]
Thanks - that seems to have done the trick!
